be/src/storage/persistent_index.h

```diff
@@ -281,6 +281,10 @@ class MutableIndex {
 
     virtual Status pk_dump(PrimaryKeyDump* dump, PrimaryIndexDumpPB* dump_pb) = 0;
 
+    virtual void set_format_version(uint32_t ver) = 0;
+
+    virtual Status completeness_check(phmap::BinaryInputArchive& ar) = 0;
```

Comment:
```suggestion
    virtual Status completeness_check(const phmap::BinaryInputArchive& ar) = 0;
```
use `const` as much as possible

Reply:
It can't be const, there is `ifstream` in `BinaryInputArchive`, and `completeness_check` will lead to state change in `ifstream`.